docs/code/plugin-introduction.rst

```diff
@@ -11,6 +11,10 @@ standard location under ``tmt/steps`` , from all directories
 provided in the ``TMT_PLUGINS`` environment variable and from
 ``tmt.plugin`` entry point.
 
+.. versionadded:: 1.35
+   You can use ``tmt.resources`` entry point to inject resource
+   files to be used for tmt, e.g. schemas or templates.
```

Comment:
@LecrisUT would it be possible to add some more documentation about this feature maybe? I.e. how to exactly use it for my plugin.

Also, according to some other comments, maybe some clashing is possible, so would it be possible to describe the best practices to prevent that?

Reply:
Added a section to it, wdyt? Feels like a plugin 101 with a step-by-step would be more appropriate, but that's a much bigger task.